src/AppInstallerCLICore/Commands/ListCommand.cpp

```diff
@@ -6,6 +6,8 @@
 #include "Workflows/WorkflowBase.h"
 #include "Resources.h"
 
+using AppInstaller::CLI::Workflow::SearchPurpose;

```

Comment:
nit: just using AppInstaller::CLI::Workflow; should be enough
And moving inside namespace AppInstaller::CLI to be with existing usings